Jenkinsfile

```diff
@@ -88,93 +88,237 @@ try {
     }
 
     stage ('Parallel Build') {
-        def variants = [
-            'coverage',
-            'clang.debug.unity',
-            'clang.debug.nounity',
-            'gcc.debug.unity',
-            'gcc.debug.nounity',
-            'clang.release.unity',
-            'gcc.release.unity'] as String[]
+        String[][] variants = [
+            ['coverage'],
+            ['docs'],
+            ['msvc.debug'],
+            ['msvc.debug', "", "PARALLEL_TESTS=false"],
```

Comment:
What is the distinction between single and double quotes in this case?  Googling about groovy indicates string versus template-able string. Is that used here?

Reply:
single quotes don't interpolate variables, doubles do. On this line, these should all be singles - I'll change that and make a pass through for any others that can be made non-interpolating. 

http://groovy-lang.org/syntax.html#all-strings